src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
```

Comment:
You write in Props that you have a user 100%
so, it's extra
![image](https://user-images.githubusercontent.com/91411692/231989935-f630a654-36e3-401a-8a4e-9f8f0382731b.png)
![image](https://user-images.githubusercontent.com/91411692/231989942-375bebfd-0156-4301-9dd2-230469e0b640.png)

```suggestion
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
```